src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/LabelPart.Fields.Edit.cshtml

```diff
@@ -0,0 +1,8 @@
+@using OrchardCore.Forms.ViewModels;
+@model LabelPartEditViewModel
+
+<fieldset class="form-group">
+    <label asp-for="For">@T["For"]</label>
+    <input asp-for="For" type="text" class="form-control content-preview-text"></input>
+    <span class="hint">@T["The ID of the form element to associate with this label."]</span>
```

Comment:
I prefer to use `id` instead of `ID` whenever we mean the attribute.

Reply:
OK

Reply:
Or would it be better to say: "The ID attribute of the..."? Since that's normal English, and still making clear that it's the ID attribute. The fact that we like to use lowercase attributes in HTML is just a convention. Technically, "ID" is valid too.

Reply:
I left this as-is, but let me know if you still think it should be renamed.